packages/format-library/src/link/inline.js

```diff
@@ -77,6 +77,39 @@ const LinkViewerUrl = ( { url } ) => {
 	);
 };
 
+const URLPopoverAtLink = ( { isActive, addingLink, value, ...props } ) => {
+	const anchorRect = useMemo( () => {
```

Comment:
Just wondering... Does this have a limit on cache size? Sounds like it should have a limit of exactly one.

Reply:
> Just wondering... Does this have a limit on cache size? Sounds like it should have a limit of exactly one.

It's not entirely clear, but it seems like something they would have considered as being the default behavior, but I also recall some similar discussion with @gziolo recently where he was showing a separate project which, by its name, might imply a different behavior 🤷‍♂️ 

https://github.com/alexreardon/use-memo-one#readme

>You may rely on useMemo as a performance optimization, not as a semantic guarantee. In the future, React may choose to “forget” some previously memoized values and recalculate them on next render, e.g. to free memory for offscreen components. Write your code so that it still works without useMemo — and then add it to optimize performance.

https://reactjs.org/docs/hooks-reference.html#usememo

>**useMemo and useCallback cache the most recent result**. However, this cache can be destroyed by React when it wants to:

Reply:
Yeah, to me it sounded like it was not the intention of `useMemo` to limit the cache to one by default. So I would consider swapping it. I'm generally wary of things that create cache without any limit. We're just taking more and more space from memory for no reason.

Reply:
> Yeah, to me it sounded like it was not the intention of `useMemo` to limit the cache to one by default. So I would consider swapping it. I'm generally wary of things that create cache without any limit. We're just taking more and more space from memory for no reason.

Oh, I was thinking the opposite by my interpretation; that yes, it would only store the one latest result in memory. I'll plan to double-check, though.

Reply:
From what I can discern, it only maintains a single value.

https://github.com/facebook/react/blob/3438e5ce879883590f68370b9ad6448a8ffdfdc1/packages/react-dom/src/server/ReactPartialRendererHooks.js#L338-L365